framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -336,8 +336,7 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
         this.visitorState = new VisitorState();
 
         this.loader = new AnnotationClassLoader(checker);
-        this.supportedQuals = createSupportedTypeQualifiers();
-        checkSupportedQuals();
+        this.supportedQuals = new HashSet<>();
```

Comment:
Shouldn't leave HashSet as a raw type.

Change all of these to HashSet<Class<? extends Annotation>>